docs/index.adoc

```diff
@@ -527,70 +545,163 @@ This diagram maps out how the `CLOSING` state transitions to the `CLOSED` state.
+|Yes 
+|`40 days`
+
+|walletClosureMinBtcBalance 
+|The minimum amount of satoshis at which a non-active wallet is eligible to begin closing. 
+|Yes 
+|`5e7 = 0.5 BTC`
+
+|walletMaxAge 
+|The age at which a non-active wallet is eligible to begin closing. 
+|Yes 
+|`26 weeks`
+
+|walletMaxBtcTransfer 
+|The amount of satoshis held by the closing wallet at which it should try to split the moved funds to multiple target wallets. 
+|Yes 
+|`10e8 = 10 BTC`
+
+4+s|Depositing
+
+|depositDustThreshold
+|The minimum amount of satoshis in a valid deposit.
 |Yes
-|`3 days`
+|`1_000_000 satoshis`
 
 4+s|Sweeping
 
-|depositTreasuryFeeDivisor
-|The deposit fee divisor of one BTC to take as a treasury fee.
-|Yes
-|`2000 => 1/2000 = 5 BPS`
+|depositTreasuryFeeDivisor 
+|A divisor used to compute the treasury fee taken from each deposit and transferred to the treasury upon sweep proof submission.
+|The deposit fee divisor of one BTC to take as a treasury fee. 
+|Yes 
+|`2_000 => 1/2_000 = 5 BPS`
 
-|depositTxMaxFee
-|The max amount of satoshis per deposit that the wallet is allowed to pay to miners.
-|Yes
-|`10000 sats`
+|depositTxMaxFee 
+|The max amount of satoshis per deposit that the wallet is allowed to pay to miners for processing the sweep transaction. 
+|Yes 
+|`100_000 satoshis`
 
 4+s|Redeeming
 
-|redemptionTreasuryFeeDivisor
-|The redemption fee divisor of one BTC to take as a treasury fee.
-|Yes
-|`2000 => 1/2000 = 5 BPS`
-
-|redemptionTimeout
-|Length of time a wallet has to fulfill a redemption.
-|Yes
-|`48 hours`
+|redemptionDustThreshold 
+|The minimum amount of satoshis in a valid redemption. 
+|Yes 
+|`1_000_000 satoshis`
+
+|redemptionTimeout 
+|The amount of time a wallet has to fulfill a redemption request. 
+|Yes 
+|`5 days`
+
+|redemptionTimeoutNotifierRewardMultiplier 
+|The percentage of the slashed stake from a redemption timeout that the notifier receives as a reward. 
+|Yes 
+|`100%`
+
+|redemptionTimeoutSlashingAmount 
+|The amount of stake in T tokens that each group member will lose if they do not fulfill a redemption request. 
+|Yes 
+|`100 * 1e18 = 100 T` 
+
+|redemptionTreasuryFeeDivisor 
+|A divisor used to compute the treasury fee taken from each redemption request and transferred to the treasury upon successful request finalization. 
+|Yes 
+|`2_000 => 1/2_000 = 5 BPS`
+
+|redemptionTxMaxFee 
+|The max amount of satoshis per redemption that the wallet is allowed to pay miners for processing the redemption transaction. 
+|Yes 
+|`100_000 satoshis`
+
+4+s|Fraud 
+
+|fraudChallengeDefeatTimeout 
+|The amount of time a wallet has to defend itself from a fraud challenge. 
+|Yes 
+|`7 days`
+
+|fraudChallengeDepositAmount 
+|The amount of ether a challenger must deposit in order to make a wallet prove its honesty. 
+|Yes 
+|`5 ether`
+
+|fraudNotifierRewardMultiplier 
+|The percentage of the slashed stake from a fraud challenge that the challenger receives as a reward. 
```

Comment:
|The percentage of the staking contract notifier reward that the fraud challenger receives as a reward if the wallet did not defeat the challenge.